packages/app/src/components/Common/Dropdown/PageItemControl.tsx

```diff
@@ -300,15 +300,19 @@ export const PageItemControlSubstance = (props: PageItemControlSubstanceProps):
     if (!isIPageInfoForOperation(presetPageInfo) && isOpen) {
       setShouldFetch(true);
     }
-    if (presetPageInfo?.isContainerFluid && !$('body').hasClass('on-search')) {
+  }, [isOpen, presetPageInfo, shouldFetch]);
+
+  // Change body class on change content width
+  const updateBodyClass = useCallback((isContainerFluid: boolean) => {
+    if (isContainerFluid && !$('body').hasClass('on-search')) {
```

Comment:
You can nest `!$('body').hasClass('growi-layout-fluid')` as well. I think it is a bit more reader-friendly. What do you think?